src/main/java/seedu/recruit/commons/util/EmailUtil.java

```diff
@@ -28,20 +30,108 @@
 import com.google.api.services.gmail.GmailScopes;
 import com.google.api.services.gmail.model.Message;
 
+import seedu.recruit.model.candidate.Candidate;
+import seedu.recruit.model.joboffer.JobOffer;
+
 /**
- * Class for using GMAIL on RecruitBook
+ * Contains variables and functions pertaining to the Email Command
  */
-public class Email {
+public class EmailUtil {
     /**
      * Global instance of the scopes required by this quickstart.
      * If modifying these scopes, delete your previously saved tokens/ folder.
      */
+    public static final String DEFAULT_FROM = "cs2113.f09.4@gmail.com";
     private static final String APPLICATION_NAME = "CS2113 F09 T04";
     private static final String TOKENS_DIRECTORY_PATH = "tokens";
     private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
     private static final List<String> SCOPES = Collections.singletonList(GmailScopes.GMAIL_COMPOSE);
     private static final String CREDENTIALS_FILE_PATH = "/credentials.json";
 
+    /**
+     * Variables for Email Command
+     */
+    private ArrayList<Candidate> candidates;
+    private ArrayList<JobOffer> jobOffers;
+    private boolean areRecipientsCandidates;
```

Comment:
why is it areRecipientsCandidates? just curious

Reply:
Oh yeah I'll probably change it to isRecipientsCandidates haha. Don't know why I gave it that name, but it stuck